pkg/sqlite/configmap_writer.go

```diff
@@ -0,0 +1,214 @@
+package sqlite
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"regexp"
+
+	"github.com/ghodss/yaml"
+	_ "github.com/mattn/go-sqlite3"
+	"github.com/sirupsen/logrus"
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/operator-framework/operator-registry/pkg/client"
+)
+
+// configmap keys can contain underscores, but configmap names can not
+var unallowedKeyChars = regexp.MustCompile("[^-A-Za-z0-9_.]")
+
+const (
+	ResourceKey                 = "operators.coreos.com.bundle.resources"
+	MediaTypeKey                = "operators.coreos.com.bundle.mediatype"
+	EnvContainerImage           = "CONTAINER_IMAGE"
+	ConfigMapImageAnnotationKey = "olm.sourceImage"
+	AnnotationsFilename         = "annotations.yaml"
+)
+
+type AnnotationsFile struct {
+	Annotations struct {
+		Resources string `json:"operators.coreos.com.bundle.resources"`
+		MediaType string `json:"operators.coreos.com.bundle.mediatype"`
+	} `json:"annotations"`
+}
+
+type ConfigMapWriter struct {
+	manifestsDir  string
+	configMapName string
+	namespace     string
+	clientset     *kubernetes.Clientset
+}
+
+var _ SQLPopulator = &ConfigMapWriter{}
```

Comment:
It's a little weird to have this object that populates configmaps conform to the interface that populates sql databases?

Reply:
Yeah - would it be better to create another interface or just drop it altogether?